TrackForce/src/test/java/com/revature/test/cuke/AssociateViewCukes.java

```diff
@@ -20,110 +21,107 @@
 
 public class AssociateViewCukes {
 	
-	//11/5/2018 Courie changed the baseURL to reference EnvManager URL instead of the system's environment URL.
-	String baseUrl = EnvManager.NGTrackForce_URL;
-	
 	@Given("^I am on the Associate Home Page$")
-	public void i_am_on_the_Associate_Home_Page() throws Throwable {
+	public void i_am_on_the_Associate_Home_Page() throws Exception {
 	System.out.println("I am on the associate homepage");
-	ServiceHooks.wait.until(ExpectedConditions.urlToBe(baseUrl + "#/associate-view"));
-		assertEquals(ServiceHooks.driver.getCurrentUrl(),baseUrl + "#/associate-view");
+	ServiceHooks.wait.until(ExpectedConditions.urlToBe(getBaseUrl() + getAssociateView()));
```

Comment:
I would recommend making the constants defined in ConstantsCukeTestUtil (e.g. ASSOCIATE_VIEW) public and using them directly. I don't think getters and setters are typically used with constants in cases like this.